frontend/packages/topology/src/components/side-bar/components/SideBarAlerts.tsx

```diff
@@ -16,14 +17,15 @@ const ResolveResourceAlerts: React.FC<{
   useResourceAlertsContent?: (element: GraphElement) => DetailsResourceAlertContent;
   element: GraphElement;
 }> = observer(function ResolveResourceAlerts({ id, useResourceAlertsContent, element }) {
-  const [showAlert, setShowAlert, loaded] = useUserSettings(
-    `${USERSETTINGS_PREFIX}.${SIDEBAR_ALERTS}.${id}.${element.getId()}`,
+  const [showAlert, setShowAlert] = useUserSettingsLocalStorage(
```

Comment:
This will cause the alerts to pop back up again after upgrade if they've been dismissed. Should we also check the config map as a fallback?

Reply:
Sure, updated the PR.